src/uu/uniq/src/uniq.rs

```diff
@@ -390,6 +419,56 @@ pub fn uu_app() -> Command {
                 .help("end lines with 0 byte, not newline")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
```

Comment:
are you sure there isn't a better way in clap ? cc @tertsdiepraam

Reply:
This is actually exactly what GNU `uniq` does too 😄 To clean this up, at the very least we can put this in a loop and they need to be hidden of course.

Reply:
See https://github.com/coreutils/coreutils/blob/95f4ee0577dd836de523f46999777fbbbe9d2772/src/uniq.c#L546-L565 😄

Reply:
And I investigated all those deprecated arguments here: https://github.com/tertsdiepraam/uutils-args/blob/main/design/arguments_in_coreutils.md#deprecated-syntax-n-and--n

Reply:
never mind, it doesn't work, since *obviously* `indices_of()` only returns the *last index* of a flag rather than all of them, now i'm tired, i'll think about this tomorrow